blockchain/txvalidator.go

```diff
@@ -116,18 +124,32 @@ func (b *BlockChain) CheckTransactionContext(blockHeight uint32, txn *Transactio
 		if err := b.checkIllegalBlocksTransaction(txn); err != nil {
 			log.Warn("[CheckIllegalBlocksTransaction],", err)
 			return ErrTransactionPayload
+		} else {
```

Comment:
No need of else switch, just `return Success`.

Reply:
Maybe someone thinks explicitly write if-else condition will be easier to read even if there are only returns expression. Well, there is no certain answer about this, modified it to uniform out code style.